### ISSUE TITLE
Post content shows excerpt even when 'More' layout has toggled the 'Hide excerpt on full content page'

### ISSUE THREAD

**Issue by jhhazelaar** [timber/timber#2218]
In Gutenberg you can add the 'More' layout block to determine the excerpt of the post. In this block there is de option 'Hide the excerpt on the full content page'. This function doesn't work.

## Expected behavior
When displaying `{{ post.content }}` and 'Hide the exceprt' is enable the excerpt needs to be stript of the content.

## Actual behavior
Excerpt is showing in `{{ post.content }}`

## Steps to reproduce behavior
Add 'More' block to a post en visit the post detail.

## What version of WordPress, PHP and Timber are you using?
Wordpress 5.3.2, Gutenberg 7.7.1, PHP 7.3.15 and Timber 1.15.2

## How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)
Composer/Packagist

**Comment by jhhazelaar** [comment]
The 'More' block with 'Hide the excerpt' in the content code is:
```
<!-- wp:more {"noTeaser":true} -->
<!--more-->
<!--noteaser-->
<!-- /wp:more -->
```
And without 'Hide excerpt'
```
<!-- wp:more -->
<!--more-->
<!-- /wp:more -->
```

Maybe split the content on `<!--noteaser-->` in de Post content function?

**Comment by jarednova** [comment]
Thanks @jhhazelaar. This is fixed in #2256
